charts/linkerd2/templates/prometheus.yaml

```diff
@@ -114,10 +114,9 @@ data:
       # drop all labels that we just made copies of in the previous labelmap
       - action: labeldrop
         regex: __meta_kubernetes_pod_label_linkerd_io_proxy_(.+)
-      # __meta_kubernetes_pod_label_linkerd_io_foo=bar =>
-      # foo=bar
+      # __meta_kubernetes_pod_label_foo=bar => foo=bar
```

Comment:
does this means that labels which previously had a `linkerd_io` prefix stripped off will now have the `linkerd_io` prefix in the label name?

Reply:
@adleong Just to get a bit more context here. What are the implications of that prefix not being stripped off?

Reply:
I'm wondering if this will cause the label names to be different, breaking existing prometheus queries

Reply:
While I believe this does cause the `linkerd_io` labels appear on the Prometheus metrics it doesn't appear to break any existing queries. I will be doing more to investigate this

Reply:
@daxmc99 Any results from the  investigation?

Reply:
@zaharidichev Sorry, I was on vacation. I will be looking at this during the week.

Reply:
Following up on this; yes this did cause the label format to change however I did not see this cause any queries to break ( I didn't see any queries that depended on this field but I might have missed one). 
I ended up doing this with plus a `labeldrop` command to remove the duplicate entries.